### PR TITLE
Blueprint: new profile comparison/management view

### PR DESCRIPTION
**Pull request by mitchell852**
## What does this PR (Pull Request) do?

- [x] This PR is related to #2391 

This PR contains a feature blueprint that proposes a new and improved profile comparison view in TP.

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
Simply read the blueprint and add your comments inline. This is only a blueprint so no documentation or tests are required.

## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR ensures that database migration sequence is correct OR this PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: blueprints/profile-param-compare-manage.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+## Proposed Change
+<!--
+*How* will this be implemented (at a high level)?
+-->
+The current view in Traffic Portal (https://tp.domain.com/#!/profiles/compare/:profile_id_1/:profile_id_2) can be redesigned to list a superset of all parameters across the 2 profiles and show their profile membership side-by-side. It can also provide the ability to remove or add parameters to each profile easily and allow the user to persist the final state of both profiles (or reset to the original state). As the user makes changes, a visible indicater can be provided (i.e. red checkboxes) to show what has been changed.
+
+![alt text](img/profile-param-compare-manage.png "Mockup of proposed profile comparison view")
```
Comment (ocket8888) [inline blueprints/profile-param-compare-manage.md]: nit: alt text on images should be meaningful. You can probably just re-use the title

Comment (mitchell852) [reply blueprints/profile-param-compare-manage.md]: @ocket8888 - are you saying change it to this?

`title`

Comment (ocket8888) [reply blueprints/profile-param-compare-manage.md]: uhh, maybe. I mean something like:

```markdown
![Mockup of proposed profile comparison view](img/profile-param-compare-manage.png "Mockup of proposed profile comparison view")
```

Comment (mitchell852) [reply blueprints/profile-param-compare-manage.md]: weird. both ways work. /shrug

Comment (mitchell852) [reply blueprints/profile-param-compare-manage.md]: btw i found the syntax here: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images

Comment (ocket8888) [reply blueprints/profile-param-compare-manage.md]: Nah, if you look at the source of the example images in that cheat sheet you'll see that they have literally `alt="alt text"` - title &#8800; alt text. Alt text is what's shown in user agents that don't support images and/or the image type of the image. It's an accessibility concern. Meanwhile:

> _"The `title` global attribute contains text representing advisory information related to the element it belongs to."_ - MDN

There's some other stuff including inheritance that differ between them, but the big difference is advisory information vs alternative representation.

**Comment by ocket8888** [comment]
License issue:
```
Error                                 Unknown! blueprints/img/profile-param-compare-manage.png
```

File: blueprints/profile-param-compare-manage.md (hunk 1 above)
Comment (mitchell852) [reply blueprints/profile-param-compare-manage.md]: fixed

**Pull request merged**